src/components/index.js

```diff
@@ -144,7 +144,7 @@ class ChatHandler {
   }
 
   getMapsAPIKey() {
-    return this.component.props.mapsApiKey || ''
+    return this.component.props.mapsApiKey || process.env.GOOGLE_MAPS_API_KEY || ''
```

Comment:
I would leave `process.env` out of this component and just pass it in in `dev/example.js`

Reply:
pass it as a parameter?

Reply:
yes in the component `mapsApiKey={process.env....}`

Reply:
in the example is run from the commandline webpack. But if used from a library in a browser `process.env` does not exist